Sources/PrimerSDK/Classes/Data Models/PrimerCheckoutQRCodeInfo.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  PrimerCheckoutQRCodeInfo.swift
+//  PrimerSDK
+//
+//  Created by Dario Carlomagno on 18/10/22.
+//
+
+#if canImport(UIKit)
+
+import Foundation
+
+// MARK: - QRCode
+
+@objc public class PrimerCheckoutQRCodeInfo: PrimerCheckoutAdditionalInfo {}
+
+// MARK: PromptPay
+
+@objc public class PromptPayCheckoutAdditionalInfo: PrimerCheckoutQRCodeInfo {
+    
+    let expiresAt: String
```

Comment:
That's a good idea 💡 .
I've just moved those classes to different files to avoid a bigger file with all the `[...]AdditionalInfo` class declarations there.

When we made the abstraction though, we did it purely to highlight the different context/domain of each _"additional info"_ object. Their purpose wasn't necessarily to adopt inheritance; especially as we can't make 💯 sure that parameters like `expiresAt` will be present in every model.